pkg/cvo/cvo.go

```diff
@@ -678,15 +678,25 @@ func (optr *Operator) sync(ctx context.Context, key string) error {
 
 	// identify the desired next version
 	desired, ok := findUpdateFromConfig(config, optr.getArchitecture())
-	if ok {
-		klog.V(2).Infof("Desired version from spec is %#v", desired)
+	stillInitializing := optr.configSync.StillInitializingFunc()()
```

Comment:
We use the name negated, can we call this `initialized` instead (the method too? To me it also seems that we don't get much value from the indirection as we call the returned func right away:

```
initialized := optr.configSync.Initialized()

if ok && initialized {
 // happy
...
```

Reply:
Renamed the function and variable.